mail/views.py

```diff
@@ -237,3 +238,34 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class MailWebhookView(APIView):
+    """
+    View class that handles HTTP requests to capture bounced email
```

Comment:
Please change this description to `View class that handles Mailgun webhooks`

Reply:
done